lms/static/js/dashboard/routers/dashboard_router.js

```diff
@@ -0,0 +1,15 @@
+;(function (define) {
+
+    define(['backbone'], function (Backbone) {
+        'use strict';
+
+        return Backbone.Router.extend({
+            routes: {
+                'dashboard/:course_tab_name': 'switchTab'
+            },
+            switchTab: function (course_tab_name) {
+                this.trigger('switchTab', course_tab_name);
+            }
+        });
+    });
+}).call(this, define || RequireJS.define);
```

Comment:
nit: please make sure that there is an empty newline at the end of file. There is a setting in `pycharm` which make sure this thing.